src/styles/utils/variables.scss

```diff
@@ -0,0 +1,16 @@
+$color-of-cell: #eee;
+$size-of-cell: 100px;
+$number-of-columns: 7;
+$gap: 1px;
+$padding: 10px;
+$duration-of-animation: 0.5s;
+$color-on-hover: #ffbfcb;
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
```

Comment:
The map keys should be quoted if they are strings, as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). This ensures consistency and prevents any potential issues with unquoted strings.